README.md

```diff
@@ -0,0 +1,6 @@
+Vulkan Capture for OBS
+======================
+
+Tools to capture Vulkan/OpenGL applications with OBS. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).
+
+For this to work you also need the corresponding OBS plugin (should be com.obsproject.Studio.Plugin.VkCapture for the OBS flatpak).
```

Comment:
```suggestion
For this to work you also need the corresponding OBS VkCapture plugin for OBS Studio. This plugin is available as [`com.obsproject.Studio.Plugin.VkCapture` on Flathub](https://github.com/flathub/com.obsproject.Studio.Plugin.VkCapture]. 
```

Reply:
I have committed the changes (don't know if GitHub notifies if I just add a commit, so I'm just gonna leave a comment)